server/src/data/users-data.js

```diff
@@ -0,0 +1,82 @@
+const _ = require('lodash');
```

Comment:
What do you think of using RamdaJs instead?

Reply:
Never used it before, it seems to be more functional style than lodash. Do you prefer RamdaJs?

Reply:
Yea. I haven't used it a whole lot, but want to move more in that direction. I prefer to stay more functional where possible.